lib/wda/webdriveragent.js

```diff
@@ -258,6 +258,7 @@ class WebDriverAgent {
       port: this.url.port,
       base: '',
       timeout: this.wdaConnectionTimeout,
+      keepAlive: true,
```

Comment:
does it make sense to make it configurable? Or you are sure there will be no buffering issues if this is set to true?

Reply:
I'm pretty sure it will be safe; it was previously enabled by default for every connection in JWProxy in general for quite some time and didn't encounter problems in this area from what I know (not in my testing environment nor did we hear of issues from users).

If we want to be safe, maybe we can make it configurable with a server argument to explicitly disable connection keepalives everywhere. What do you think?

Reply:
I would say we add a server argument `--disable-keepalive` and enable `forever` by default

Reply:
@dpgraham @imurchie Any thoughts?

Reply:
keepalive _is_ disabled by default, in `appium-base-driver`. This is only for the connection to WDA. Since we want different behavior on different proxy connections, a server flag would not be useful.

Reply:
> we want different behavior on different proxy connections

do we?

Reply:
That was what prompted all this work by @Dan-Maor, yes. https://github.com/appium/appium/issues/12599#issue-441311364

Reply:
this also must be the case for UIA2/Espresso, which also use jwproxy

Reply:
That'll have to be experimented with.